src/esp/metadata/managers/AbsObjAttributesManagerBase.h

```diff
@@ -0,0 +1,297 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef ESP_METADATA_MANAGERS_ABSOBJATTRIBUTESMANAGERBASE_H_
+#define ESP_METADATA_MANAGERS_ABSOBJATTRIBUTESMANAGERBASE_H_
+
+/** @file
+ * @brief Class Template @ref
+ * esp::metadata::managers::AbsObjAttributesManager
+ */
+
+#include "esp/metadata/attributes/ObjectAttributes.h"
+
+#include "esp/metadata/managers/AttributesManagerBase.h"
+
+namespace Cr = Corrade;
+
+namespace esp {
+namespace metadata {
+namespace managers {
+namespace Attrs = esp::metadata::attributes;
+/**
+ * @brief Class template defining responsibilities and functionality for
+ * managing @ref esp::metadata::attributes::AbstractObjectAttributes constructs.
+ * @tparam T the type of managed attributes a particular specialization
+ * of this class works with.  Must inherit from @ref
+ * esp::metadata::attributes::AbstractObjectAttributes.
+ */
+template <class T>
+class AbsObjAttributesManager : public AttributesManager<T> {
```

Comment:
Err... `Abs` as in "abstract"? at first I thought it's something "absolute", because that's what `abs` is commonly used for. Would it hurt if it would be ~~`AbsoluteObjectAttributesManager`~~ `AbstractObjectAttributesManager` instead? :sweat_smile:

Reply:
Ok, just seemed ...long.  But i'm cool with that.